asciinema2gif

```diff
@@ -5,18 +5,18 @@
 # Halt on error.
 set -e
 
-rundir=$PWD
+readonly rundir="$(basename "${0}")"
```

Comment:
Done (there are shorter ways, with `realpath`, but those don’t work out-of-the-box with OS X).